ts/modules/series-label.src.ts

```diff
@@ -1032,25 +1067,23 @@ Chart.prototype.drawSeriesLabels = function (): void {
                         };
                         attr = {};
                     }
-                    (series.labelBySeries as any)
+
+                    // Default initial animation to a fraction of the series
+                    // animation (#9396)
+                    let animationOptions: Highcharts.AnimationOptionsObject|undefined;
+                    if (isNew) {
+                        animationOptions = animObject(series.options.animation);
+                        (animationOptions.duration as number) *= 0.2;
```

Comment:
This kind of casting (`as number`) should be avoided, as it hides undefined/null problems.
- If duration is always set, declaration typing of the property should be changed instead.
- If optional, do `animationOptions.duration = (animationOptions.duration || 0) * 0.2` instead.
- If you can not deal with it now, cast it to `any` and we will pick it up later.

Reply:
If (or when) #13005 will be merged, then `animObject()` will always return `{ duration, defer }` and casting to `any` will be unnecessary. So I would go for now with the third option :)

Reply:
Fantastic! 👍

Reply:
Great!